src/cli/pack_command.zig

```diff
@@ -2302,7 +2302,7 @@ pub const PackCommand = struct {
     };
 
     pub const IgnorePatterns = struct {
-        list: []const Pattern,
+        list: []Pattern,
```

Comment:
```suggestion
        list: []const Pattern,
```

and make `CowSlice.deinit` take a `*const Self`

Reply:
it's not setting it (other than the constCast -> memset undefined in free)